### PR TITLE
create Layout stars

### PR DESCRIPTION
**Pull request by KarelVito**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url(./images/star.svg);
```
Comment (SanyaBratashchuk) [inline src/style.css:41]: combine selectors with same styles. 
Example `.block1, .block2, .block3 {  color: white;}`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
```
Comment (SanyaBratashchuk) [inline src/style.css:21]: Set is as default background-image for .stars__star

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: readme.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://KarelVito.github.io/layout_stars/)
+- [TEST REPORT LINK](https://KarelVito.github.io/layout_stars/report/html_report/)
```
Comment (ivbutsykin) [inline readme.md:4]: Don't forget add it in description

**Review by ivbutsykin — APPROVED** [review]
Good job!

**Comment by KarelVito** [comment]
- [DEMO LINK]
- https://KarelVito.github.io/layout_stars/
- [TEST REPORT LINK]
- https://KarelVito.github.io/layout_stars/report/html_report/

**Review by SerhiiKirik — APPROVED** [review]
Sync 🧑‍💻

**Pull request closed without merging**